public/index.js

```diff
@@ -139,6 +141,68 @@ async function initMap()
         
         map.controls[google.maps.ControlPosition.BOTTOM_LEFT].push(toggleStopsButtonDiv);
         loading = false;
+
+        let toggleAlertsButton = document.createElement('button');
+        toggleAlertsButton.id = 'toggleAlertsButton';
+        toggleAlertsButton.textContent = 'View Alerts';
+        toggleAlertsButton.classList.add(CSS_CLASS_TOGGLE_STOPS_BUTTON);

```

Comment:
If this CSS class is going to be shared can we change the name to something just like `CSS_CLASS_BUTTON`?